index.bs

```diff
@@ -1175,7 +1175,7 @@ const intermediateOutput2 = builder.add(constant2, input2);
 const output = builder.mul(intermediateOutput1, intermediateOutput2);
 
 // Create the model by identifying the outputs.
-const model = builder.createModel({'output', output});
+const model = builder.createModel({output});
```

Comment:
syntax error

Reply:
I use the [shorthand property names (ES2015)](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer)